<!-- Kevin Daniel
	June 29, 2018
  -->

<!-- Launch file to begin QR detection -->

<launch>
	<!-- Start cam_bridge -->
	<node pkg = "image_processing" type = "cam_bridge.py" name = "cam_bridge" output = "screen" respawn = "true">
		<rosparam command = "load" file = "$(find image_processing)/config/cam_bridge_params.yml"/>
	</node>

	<!-- Start the image preprocessor-->
	<node pkg = "image_processing" type = "img_preprocessing.py" name = "img_preprocessing" output = "screen" respawn = "true">
		<rosparam command = "load" file = "$(find image_processing)/config/img_preprocessing_params.yml"/>
		<param name = "path to corners" value = "$(find image_processing)/config"/>
	</node>

	<!-- Start the QR detector-->
	<node pkg = "image_processing" type = "qr_detector.py" name = "qr_detector" output = "screen" respawn = "true">
		<rosparam command = "load" file = "$(find image_processing)/config/qr_detector_params.yml"/>
	</node>
</launch>


